src/main/java/gov/usds/case_issues/db/model/TroubleCase.java

```diff
@@ -39,16 +39,29 @@
 @NamedNativeQueries({
 	@NamedNativeQuery(
 		name = "snoozed",
-		query = "SELECT * from " + TroubleCase.CASE_DTO_CTE
-			  + "WHERE last_snooze_end >= CURRENT_TIMESTAMP "
-			  + "ORDER BY last_snooze_end ASC, case_creation ASC, internal_id ASC",
+		query = "SELECT * FROM " + TroubleCase.CASE_DTO_CTE
+				+ "WHERE last_snooze_end >= CURRENT_TIMESTAMP "
+				+ "ORDER BY last_snooze_end ASC, case_creation ASC, internal_id ASC "
+				+ "LIMIT :size",
+		resultSetMapping="snoozeCaseMapping"
+	),
+	@NamedNativeQuery(
+		name = "snoozedAfter",
+		query = "SELECT * FROM " + TroubleCase.CASE_DTO_CTE
+				+ "WHERE last_snooze_end >= CURRENT_TIMESTAMP "
+				+ "AND last_snooze_end >= :lastSnoozeEnd "
+				+ "AND case_creation >= :caseCreation "
+				+ "AND internal_id >= :internalId "
```

Comment:
Final tie-break comparison should be strictly-greater-than, no?

I think #61 is about this, maybe?